src/main/res/menu/group_create.xml

```diff
@@ -2,7 +2,7 @@
 
 <menu xmlns:android="http://schemas.android.com/apk/res/android" xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <item android:title="@string/group_create_button"
+    <item android:title="@string/perm_continue"
```

Comment:
this was completely wrong, always showing "create group" also for broadcast/channel creation, and when editing existing group/broadcast was also saying "create group" which is wrong since it would be about saving changes to group name/avatar etc. for now I just changed it to a neutral string "continue" that fits for all, but if provided we could have more fancy string adaption for every situation